### DISCUSSION TITLE
Option to enable scenario completion progress from past saves?

### DISCUSSION THREAD

**Discussion by terrytek12** [OpenRCT2/OpenRCT2#20712] [Feature requests]
This one might be more of an oversight, but I thought about what if OpenRCT2 allowed the import of saves from previous copies of RCT1/2 and also could transfer the completion data over into your current version, so that if you have the "all scenarios unlocked" cheat off and play it the rct1 way of unlocking, you can use past saves for completion progress instead of having to start all over again. And it would only have to migrate the completion data once; say, for example, if there's multiple saves of a scenario that are completed it would probably do the most recent instance of completion or something like that, and it can ignore saves for scenarios that you have not unlocked yet.  There might also have that aspect where people wouldn't mind starting all over again, but for those that have already completed the scenarios and still have the saves would rather just continue where they left off.

**Comment by MASHzj** [comment]
You don't have to start all over again. You can open sv4 and sv6 files and use the "win scenario" cheat. That's what I did with scenarios where I forgot to save just before winning. But this has the consequence that the company value (which is displayed in the scenario list) may be much higher than at the moment that I completed the scenario. So if this would become a more automatic approach like you suggest, my question is what are you going to display as company value in such cases?
